source/includes/_files.md

```diff
@@ -0,0 +1,208 @@
+```
+
+> Example response
+
+```json
+{
+    "file_id": "cheese_data.csv",
+    "url": "odsfile://cheese_data.csv",
+    "filename": "Cheese Data.csv",
+    "properties": {
+        "mimetype": "text/csv"
+    },
+    "created": "2017-06-07T15:16:05.701266+00:00"
+}
+```
+
+
+## Upload a file
+
+This endpoint is for uploading a new file to the platform.
+There are two ways to send files to the platform: either by sending a full file using the HTTP multipart mechanism, or by sending the file content and other metadata such as its type and file name. The use case dictates which method works best. For instance, in order to create files out of batch processes it might be easier to use the content method, but to upload an existing file from a local hard drive, the multipart method might work better. Both methods have the same result: they create a file on the domain.
+
+> Definition
+
+```HTTP
+POST https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/files/
+```
+
+This endpoint can receive a file sent in multipart, or file content sent through the content POST parameter, along with a mimetype and an optional filename.
+
+### Multipart file upload
+
+The multipart file upload is an easy way to send a file with a HTML form. To do so, an example upload form is shown. Note that the name of the parameter is `file`.
+
+> Example HTML file upload form
+
+```html
+<form action="https://yourdomain.opendatasoft.com/api/management/v2/files/" method="post" enctype="multipart/form-data">
+    <input type="file" name="file">
+    <input type="submit">
+</form>
+```
+
+> Example multipart request
+
+```shell
+curl -XPOST 'https://yourdomain.opendatasoft.com/api/management/v2/files/' \
+    -F file=@data.csv
+```
+
+Parameter | Description
+--------- | -----------
+`file` <br> **POST parameter** <br> *file* | Multipart file to upload. This is a POST parameter
+
+### POST content upload
+
+> Example POST content request
+
+```shell
+curl -XPOST 'https://yourdomain.opendatasoft.com/api/management/v2/files/' \
+    -d '{"content": "language,phrase\nEnglish,Hello World\Esperanto,Saluton mondo\n", "mimetype": "text/csv", "filename": "data.csv"}
+```
+
+Parameter | Description
+--------- | -----------
+`content` <br> *string* | Content of the file to create
+`mimetype` <br> *string* | Mimetype of the file
+`filename` <br> *string* | **optional** Name of the file. Defaults to `file`
+
+
+### Response
+> Example response
+
+```json
+{
+    "file_id": "data.csv",
+    "url": "odsfile://data.csv",
+    "filename": "data.csv",
+    "properties": {
+        "mimetype": "text/csv"
+    },
+    "created": "2017-06-07T15:16:05.701266+00:00"
+}
+```
+
+On success, an HTTP 200 is returned along with the file object of the uploaded file.
+
+
+## Download a file
+
+This endpoint is for downloading a file with the provided file_id from the platform.
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/files/download_file/{FILE_ID}
```

Comment:
Documentation does not match the code.
In the code, the url is: /api/management/v2/download_file/{FILE_ID} (without the files)

Reply:
Ah, yes :grimacing: 

:+1: